core/shared/src/main/scala/fs2/concurrent/Queue.scala

```diff
@@ -46,6 +46,16 @@ trait Dequeue1[F[_], A] {
   def tryDequeue1: F[Option[A]]
 }
 
+/** Provides the ability to dequeue individual chunks from a `Queue`. */
+trait DequeueChunk1[F[_], A] {
+
+  /** Dequeues one `Chunk[A]` with no more than `maxSize` elements. Completes once one is ready. */
+  def dequeueChunk1(maxSize: Int): F[Chunk[A]]
+
+  /** Tries to dequeue a single chunk of no more than `max size` elements; yields to `None` if the element cannot be dequeued. */
```

Comment:
> if the element cannot be dequeued

When does this happen?

Reply:
If there's no element available at the time of the `tryDequeueChunk1` call. Instead of semantically blocking for a chunk, it returns immediately with a None.

Reply:
That wording would be more clear in the doc, for me at least. Thanks!